platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/preconsensus/SyncPreConsensusEventWriter.java

```diff
@@ -331,12 +346,15 @@ private void closeFile() {
      * Calculate the generation span for a new file that is about to be created.
      */
     private long computeNewFileSpan(final long minimumFileGeneration, final long nextGenerationToWrite) {
-        if (averageGenerationalSpanUtilization.isEmpty()) {
-            return bootstrapGenerationalSpan;
-        }
 
-        final long desiredSpan =
-                (long) (averageGenerationalSpanUtilization.getAverage() * generationalSpanOverlapFactor);
+        final long basisSpan = (bootstrapMode || averageGenerationalSpanUtilization.isEmpty())
```

Comment:
Is it conceivable that `bootstrapMode` could be false, *and* `averageGenerationalSpanUtilization` could be empty?

Reply:
Yes, I uncovered this case when testing. It will be empty exactly once, the first time it is called after we exit bootstrap mode.